### PR TITLE
Stories/translations interface 147771003

### PR DESCRIPTION
**Pull request by everestwitman**

### CODE REVIEW

File: app/views/admin/translation_keys/edit.html.haml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+- new = @translation_key.new_record?
```
Comment (eanders) [inline app/views/admin/translation_keys/edit.html.haml]: Do we need this file anymore?

File: config/initializers/fast_gettext.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# db access is cached <-> only first lookup hits the db
+require "fast_gettext/translation_repository/db"
+FastGettext::TranslationRepository::Db.require_models #load and include default models
+
+FastGettext.add_text_domain(
+  'hmis_warehouse',
```
Comment (eanders) [inline config/initializers/fast_gettext.rb]: Is this the database name?  If it is, it needs to come from the database.yml file.

File: config/routes.rb [hunk 3]
```diff
@@ -228,9 +228,11 @@ def healthcare_routes
       end
       resources :roles, only: [:index]
     end
+    resources :translation_keys
```
Comment (eanders) [inline config/routes.rb]: I think we can significantly limit the routes here, maybe only: [:index, :update]?

File: db/migrate/20170627154145_create_translation_tables.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
+class CreateTranslationTables < ActiveRecord::Migration
+  def self.up
+    create_table :translation_keys do |t|
+      t.string :key, :unique=>true, :null=>false
+      t.timestamps
+    end
+    add_index :translation_keys, :key #I am not sure if this helps....
+
+    create_table :translation_texts do |t|
+      t.text :text
```
Comment (eanders) [inline db/migrate/20170627154145_create_translation_tables.rb]: Let's make this `null: false, default: ''`

File: lib/tasks/sync_po_to_db.rake [hunk 5]
```diff
@@ -0,0 +1,40 @@
+desc "synchronise po files with db, creating keys and translations that do not exist"
+task :sync_po_to_db => :environment do
+  folder = ENV['FOLDER']||'locale'
+
+  gem 'grosser-pomo', '>=0.5.1'
```
Comment (eanders) [inline lib/tasks/sync_po_to_db.rake]: This looks odd.  Do you not bring this in in the Gemfile?

File: locale/fr/hmis_warehouse.edit.po [hunk 6]
```diff
@@ -0,0 +1,28 @@
+msgid ""
```
Comment (eanders) [inline locale/fr/hmis_warehouse.edit.po]: Let's remove the fr translations from the repository

File: app/views/admin/translation_keys/edit.html.haml (hunk 1 above)
Comment (everestwitman) [inline app/views/admin/translation_keys/edit.html.haml]: Nope. Should I remove the edit function in the controller then?

File: config/initializers/fast_gettext.rb (hunk 2 above)
Comment (everestwitman) [inline config/initializers/fast_gettext.rb]: No. I think that this is just where the name of the app goes.

**Pull request merged**